caps/eth.md

```diff
@@ -1,49 +1,52 @@
 # Ethereum Wire Protocol (ETH)
 
 'eth' is a protocol on the [RLPx] transport that facilitates exchange of Ethereum
-blockchain information between peers. The current protocol version is **eth/64**. See end
+blockchain information between peers. The current protocol version is **eth/65**. See end
```

Comment:
Actually, although we haven't implemented in in geth, AFAIK `eth66` is the most recent approved protocol

Reply:
See also https://github.com/ethereum/EIPs/issues/2482 and https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2481.md

Reply:
Yes, that's correct, but the version here is the *spec version* and this PR updates it to eth/65.